inputs/wysiwyg.vue

```diff
@@ -695,6 +695,10 @@
         }
       });
 
+      if (this.schema.disabled) {
```

Comment:
Why are we taking it from `schema` here, and not from `props` as the others?

Reply:
I didn't want anyone to confuse it with the `isDisabled` data prop being used in the same component